internal/nginx-meshctl/commands/services.go

```diff
@@ -140,18 +135,5 @@ func isNamespaceInjectionEnabled(ctx context.Context, k8sClient client.Client, n
 		fmt.Printf("error getting namespace: %v\n", err)
 		return false, err
 	}
-	return slices.Contains(*meshConfig.EnabledNamespaces, ns) || nsObj.GetLabels()[mesh.AutoInjectLabel] == mesh.AutoInjectionEnabled, nil
-}
-
-// getMeshConfig fetches the mesh.MeshConfig of the mesh using the mesh.MeshClient.
-func getMeshConfig() (*mesh.MeshConfig, error) {
-	meshClient, err := mesh.NewMeshClient(initK8sClient.Config(), meshTimeout)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get mesh client: %w", err)
-	}
-	meshConfig, err := GetMeshConfig(meshClient)
-	if err != nil {
-		return nil, fmt.Errorf("unable to get mesh config: %w", err)
-	}
-	return meshConfig, nil
+	return nsObj.GetLabels()[mesh.AutoInjectLabel] == mesh.AutoInjectionEnabled, nil
```

Comment:
We might want to add back a check that `nsObj.name` is not in ignored namespaces, since we decided that even if customers label an ignored namespace we still shouldn't inject that namespace (so if a service is in an ignored namespace it's not 'part of the mesh' even if it's labeled).

Reply:
Added a check to return `false` if the namespace is an ignored namespace.